tests/duff.c

```diff
@@ -0,0 +1,27 @@
+
+void copy(int *to, int *from, int count)
```

Comment:
Use `clang-format` to indent `duff.c`

Reply:
However, the bootstrapped `amacc` failed to build `duff`.

Reply:
I don't have clang-format on my system.

Reply:
Have you tried it?  I would expect clang-format to mangle that code.

Reply:
Have you tried it?  I would expect clang-format to mangle that code.